examples/docker-compose/README.md

```diff
@@ -0,0 +1,85 @@
+This folder contains files and sub-folders that demonstrate how to run docker-stack notebook containers using Docker Compose on a Docker Machine-controlled host.
+
+## Pre-requisites
+
+* [Docker Engine](https://docs.docker.com/engine/) 1.10.0+
+* [Docker Machine](https://docs.docker.com/machine/) 0.6.0+
+* [Docker Compose](https://docs.docker.com/compose/) 1.6.0+
+
+See the [installation instructions](https://docs.docker.com/engine/installation/) for your environment.
+
+## Provision Docker machines
+
+### Provision a VirtualBox VM on local desktop
+
+```
+bin/vbox.sh mymachine
+```
+
+### Provision a virtual device on IBM SoftLayer
+
+```
+export SOFTLAYER_USER=my_softlayer_username
+export SOFTLAYER_API_KEY=my_softlayer_api_key
+export SOFTLAYER_DOMAIN=my.domain
+
+# Create virtual device
+bin/softlayer.sh myhost
+
+# Add DNS entry (SoftLayer DNS zone must exist for SOFTLAYER_DOMAIN)
+bin/sl-dns.sh myhost
+```
+
+## Deploy stand-alone Jupyter Notebook
+
+Build and run a `jupyter/minimal-notebook` container on an existing Docker machine.
```

Comment:
I can add an environment variable to allow users to specify a different image or Dockerfile.  Of course, there are limitations.  Changing the image will only work if it's a notebook image; otherwise, the compose files (commands, env vars) would be broken.